sparse/impl/KokkosSparse_gauss_seidel_impl.hpp

```diff
@@ -825,6 +825,7 @@ class PointGaussSeidel {
   void initialize_symbolic() {
     auto gsHandle                    = get_gs_handle();
     const size_type longRowThreshold = gsHandle->get_long_row_threshold();
+    const MyExecSpace my_exec_space  = gsHandle->get_execution_space();
```

Comment:
By default, this execution space is not partitioned. The implementation is not aware of the number of streams, though.